desparchado/templates/500.html

```
@@ -1,15 +1,30 @@
 {% extends 'layout/base.html' %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)